aws-traffic-mirroring/templates/vpc-mirroring-target-instance.yaml

```diff
@@ -19,7 +19,7 @@ Parameters:
   SnifferVersion:
     Type: String
     Description: Seekret Sniffer Version
-    Default: 1
+    Default: 2
```

Comment:
you need to update the configuration and the command line

Reply:
which cfg?

Reply:
fixed